packages/vscode/src/plugins/prisma-postgres-manager/PrismaPostgresRepository.ts

```diff
@@ -591,4 +627,45 @@ export class PrismaPostgresApiRepository implements PrismaPostgresRepository {
       this.refreshEventEmitter.fire(),
     )
   }
+
+  getLocalDatabases() {
```

Comment:
There are a bunch of synchronous file system calls in here. Am not an expert in how far VSCode decouples the actual UI from what code we run here, but usually best practice is to not perform synchronous IO on the main UI thread. => Let's maybe use async file access methods here?

Esp. considering that we have no in memory cache here but any call to `getLocalDatabases` performs all this IO freshly.

Reply:
Good point, will fix